examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,54 @@
+"""
+===========================
+WCS Axes Plotting
```

Comment:
`WCS Axes Plotting` title is a bit too broad, as this example is just showing how to plot points on a map  
```suggestion
Plotting points on a Map with WCSAxes
```